### ISSUE TITLE
recommended fees not even close to correct rate

### ISSUE THREAD

**Issue by davestaxcp** [jdogresorg/freewallet-desktop#183]
just tested a subasset token mint with an easyasset description that showed a $3.50 recommended fee 

the address started with a clean 1 input address and I chose a $12 fee (3-4x recommended) and the broadcasted tx came out as 44.8 sat/vb with a 17.9 effective sat/vb fee rate

this is the example tx: https://mempool.space/tx/49d3282c008cd0328e91a506cd1a256c61376107fae34f48f6769c0bd0fd9c51#flow=&vout=4

Freewallet is not estimating fees correctly for asset description and token minting

the small multi-sig outputs looks like the token minting and description, but the last output was not calculated correctly in the UI .... even when I just had to guess 3x or 4x the fee

I suspect false estimations of fees in the Freewallet UI exist at the moment for other features like 

- minting named assets
- changing description for a token
- multi-sends (especially large ones)

@jdogresorg  you stated "Freewallet fees are based on a tx with one input n 2 outputs… so yeah, not accurate for larger txs"

this is causing an extreme amount of issues for all types of users, experienced or not

at the moment this wallet is basically unusable

used v0.9.26

**Comment by jdogresorg** [comment]
1. already an issue open for fee estimation here https://github.com/jdogresorg/freewallet-desktop/issues/23

2. Freewallet has not changed anything in the new version, so this is the EXACT same way the wallet has always functioned... the whole "wallet is basically usable" is a strange statement... cuz nothing has changed, and ppl using it all the time.

3. As has been stated many times, I will work on fee estimation when I have some time... this is an OPEN SOURCE project so anyone is free to contribute code, or to create bounties to get certain features in the wallet done.... But, if it is just more of the same "we need this, we gonna just keep screaming, but not pay for features"... well.. I'm over that game.

TLDR, I will work on fee estimation in the wallet when I get a chance... unfortunately, I am only one person, and can only work on so much.

**Comment by jdogresorg** [comment]
Speaking with Javier now about paying him out of my own pocket to work on the fee estimation... cuz I'm too busy.

**Comment by jdogresorg** [comment]
fee estimation updated in latest release d48fce1
